dogs.html

```diff
@@ -0,0 +1,167 @@
+
+
+<!DOCTYPE html>
+<html lang="zxx">
+  <head>
+    <meta charset="utf-8">
+    <title>Adopt a Dog!</title>
+    <link href="https://fonts.googleapis.com/css?family=Courgette" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Work+Sans:700" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Work+Sans:400" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Arbutus+Slab" rel="stylesheet">
+    <link rel="stylesheet" type="text/css" href="styles.css">
+  </head>
+  <body>
+
+      <header>
+        <img class="header-logo" src="images/adoptadog-logo.svg" alt="Adopt a Dog logo">
+        <h1>Seattle's premier dog adoption agency</h1>
+        <img class="cart-img" src="images/cart-icon.svg" alt="Cart icon">
+        <a href="#" class="checkout-button">Checkout</a>
+      </header>
+
+      <nav>
+        <a href="index.html">Home&nbsp;&nbsp;/</a>
+        <a href="dogs.html">&nbsp;&nbsp;Dogs&nbsp;&nbsp;/</a>
+        <a href="blog.html">&nbsp;&nbsp;Blog</a>
+      </nav>
+
+        <h2>Here are all of our dogs!</h2>
+
+        <div class="row">
```

Comment:
I get what you're going to do with these classes of row and column. Maybe there are better more semantic words you could use?